cli/src/cli.rs

```diff
@@ -2159,6 +2162,7 @@ mod tests {
             },
             amount: SpendAmount::Some(30),
             sign_only: false,
+            force: false,
```

Comment:
I did, but the result fails because the tests bypass the CLI part of the code and go straight into the CliCommand::CreateStakeAccount which sits after the force check, then it fails with a program error, if I'm understanding correctly. So I removed that test again